contrib.php

```diff
@@ -39,7 +45,7 @@ function multidimsort($array_in, $column)
     reset($multiarray);
 
     // traverse new array of index values and add the corresponding element of the input array to the correct position in the output array
-    while (list ($key, $val) = each ($multiarray))
+    while (list ($key, $val) = arrayEach ($multiarray))
```

Comment:
why not simply use a foreach loop here?

Reply:
last remaining comment before merging- thanks for your patience ;)

Reply:
fixed